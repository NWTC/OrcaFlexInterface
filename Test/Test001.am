## This file was generated by Orca_Driver (v1.01.00a-adp, 22-Apr-2015) on 02-Sep-2015 at 10:09:40.
## This file contains the added mass matrix that is returned from OrcaFlex
## It is arranged in a 6x6 matrix
# 
Added Mass values (kg, kg-m, kg-m^2):

    Dimension     TDxi        TDyi        TDzi        RDxi        RDyi        RDzi  
   ---------------------------------------------------------------------------------
    TDxi          0.00        0.00        0.00        0.00        0.00        0.00     
    TDyi          0.00        0.00        0.00        0.00        0.00        0.00     
    TDzi          0.00        0.00        0.00        0.00        0.00        0.00     
    RDxi          0.00        0.00        0.00        0.00        0.00        0.00     
    RDyi          0.00        0.00        0.00        0.00        0.00        0.00     
    RDzi          0.00        0.00        0.00        0.00        0.00        0.00     
