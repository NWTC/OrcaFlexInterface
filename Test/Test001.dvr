Test number 001
OrcaFlexWrapper driver input file.  V1.00
   t           echo
----  OrcaFlex inputs  -------------------------------------------------------
   0.0025      DT                -- timestep size for driver to take (or DEFAULT for what the file contains)
   OrcaFlex_Interface.dat     OrcaIptFileName   -- name of OrcaFlex interface input file (relative, or including the full path) -- no file extension
----  Coordinates  -----------------------------------------------------------
   t           PtfmCoords        -- We are specifying the platform coordinates
   t           Degrees           -- Rotations specified in degrees and degrees/sec (radians otherwise)
 1,0,0,0,5,0   PtfmCoord         -- X,Y,Z,R1,R2,R3 coordinate and rotation       (m,      degrees)
 0,0,0,0,0,0   PtfmVeloc         -- Velocity (time derivative of PtfmCoord)      (m/s,    degrees/sec)
 0,0,0,0,0,0   PtfmAccel         -- Accelity (2nd time derivative of PtfmCoord)  (m/s^2,  degrees/sec^2)
   t           AddedMass         -- Write the added mass matrix to the screen
   t           AddedMassFile     -- Write the added mass matrix to a file (.am)
----  Points file input (output given as POINTSFILENAME.Velocity.EXT)  -------
   t           PointsFile        -- read in a list of points from a file
   t           PointsDegrees     -- Angles are specified in degrees in the points file
"Test001--points.txt"   PointsFileName -- name of points file (comma separated coordinate and velocity set: x,y,z,R1,R2,R3,Vx,Vy,Vz,Vr1,Vr2,Vr3 # symbol for comments)
END of driver input file
