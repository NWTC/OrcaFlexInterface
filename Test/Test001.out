## This file was generated by Orca_Driver (v1.01.00a-adp, 22-Apr-2015) on 02-Sep-2015 at 10:09:40.
# 
#       TDxi             TDyi             TDzi             RDxi             RDyi             RDzi             TVxi             TVyi             TVzi             RVxi             RVyi             RVzi             Fxi              Fyi              Fzi              Mxi              Myi              Mzi
#       (m)              (m)              (m)              (deg)            (deg)            (deg)            (m/s)            (m/s)            (m/s)            (deg/s)          (deg/s)          (deg/s)          (kN)             (kN)             (kN)            (kN m)           (kN m)           (kN m)
     1.0000000        0.0000000        0.0000000        0.0000000        4.9999995        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000        0.0000000
